.github/pages/index.html

```diff
@@ -0,0 +1,157 @@
+<html lang="en">
+
+<head>
+    <title>Configurable Systems</title>
+    <link rel="icon" type="image/x-icon" href="https://www.se.cs.uni-saarland.de/images/se-icon.png">
+    <link href="index.css" rel="stylesheet" type="text/css">
+    <script type="text/javascript">
+        const search = new URLSearchParams(window.location.search);
+        const domain = search.has('domain') ? search.get('domain') : 'https://se-sic.github.io/ConfigurableSystems';
+
+        function render() {
+            fetch(`${domain}/index.json`)
+                .then(response => response.json())
+                .then(index => render_index(index))
+                .catch(console.error);
+        }
+    </script>
+</head>
+
+<body>
+<div id="content">
+    <div id="index">
+        <form>
+            <label for="filter">
+            </label>
+            <input id="filter" type="text" class="input" placeholder="filter..."/>
+            <script>
+                document.getElementById('filter').addEventListener('input', (event) => {
+                    const input = event.target.value.trim().toLowerCase();
+                    document.getElementById('table').querySelectorAll('.item').forEach(item => {
+                        const data = item.querySelector('.data.name');
+                        if (!input || data.textContent.toLowerCase().includes(input)) {
+                            item.className = 'item default';
+                            data.innerHTML = data.textContent.replace(new RegExp(input, 'gi'), (match) => {
+                                return `<mark>${match}</mark>`;
+                            });
+                        } else {
+                            item.className = 'item hidden';
+                            data.innerHTML = data.textContent;
+                        }
+                    });
+                });
+            </script>
+            <button id="clear" type="button">
+                x
+            </button>
+            <script>
+                document.getElementById('clear').addEventListener('click', (_) => {
+                    document.getElementById('filter').value = '';
+                    document.getElementById('table').querySelectorAll('.item').forEach(item => {
+                        item.className = 'item default';
+                        const data = item.querySelector('.data.name');
+                        data.innerHTML = data.textContent;
+                    });
+                });
+            </script>
+        </form>
+        <table>
+            <colgroup>
+                <col style="width:25%">
+                <col style="width:20%">
+                <col style="width:20%">
+                <col style="width:30%">
+                <col style="width:5%">
+            </colgroup>
+            <thead>
+            <tr>
+                <th>Name</th>
+                <th>Metadata</th>
+                <th>Files</th>
+                <th>Preview</th>
+                <th>Links</th>
```

Comment:
```suggestion
                <th>Edit</th>
```
Maybe we should call this edit, as it opens the gh editor?

Reply:
This column may provide, for example, links to the original project source and so on.

Reply:
ok, then let's keep the old wording